src/pages/services/ledger/create-token-modal.tsx

```diff
@@ -148,22 +144,26 @@ export function CreateTokenModal({
             </FormControl>
           </GridItem>
           <GridItem colSpan={5}>
-            <FormControl>
-              <HStack>
-                <FormLabel htmlFor="address">Destination Address</FormLabel>
-                <Popover trigger="hover">
-                  <PopoverTrigger>
-                    <Icon as={FiInfo} />
-                  </PopoverTrigger>
-                  <PopoverContent bg="brand.teal.700" color="white" p={3}>
-                    <PopoverBody>
-                      The destination address is the current user and cannot be
-                      changed at this time.
-                    </PopoverBody>
-                  </PopoverContent>
-                </Popover>
-              </HStack>
-              <Input fontFamily="mono" isDisabled value={address} />
+            <FormControl isInvalid={!!errors.destination}>
+              <FormLabel htmlFor="destination">Destination Address</FormLabel>
+              <Controller
+                name="destination"
+                control={control}
+                rules={{
+                  required: true,
+                  validate: {
+                    isManyAddress: (v) => new RegExp(/^m\w{24,}$/).test(v),
```

Comment:
A more accurate regex for detecting a valid Many address is welcome if we have one (I looked in the spec and `many-rs`).